bi-connectors/PowerBIConnector/OpenSearch.md

```diff
@@ -0,0 +1,67 @@
+#  Connecting OpenSearch to Microsoft Power BI Desktop
+
+>[!Note]
+
+>The following connector article is provided by OpenSearch, the owner of this connector and a member of the Microsoft Power Query Connector Certification Program. If you have questions regarding the content of this article or have changes you would like to see made to this article, visit the OpenSearch website and use the support channels there.
+
+## Summary
+| Item | Description |
+| ---- | ----------- |
+| Release State | General Availability |
+| Products | Power BI Desktop |
+| Authentication Types Supported | Basic |
+| | |
+
+## Prerequisites
+* Microsoft Power BI Desktop
+* [OpenSearch](https://docs-beta.opensearch.org/opensearch/install/index/)
+* [OpenSearch SQL ODBC driver](https://docs-beta.opensearch.org/search-plugins/sql/odbc/)
+
+## Capabilities supported
+* Import
+* DirectQuery
+
+## Connect to OpenSearch
+1. Open Power BI Desktop.
+
+2. Click on **Home** > **Get Data** > **More** > **Other**. Select **OpenSearch**. Click on **Connect**.
+
+<img src="img/pbi_select_connector.png" width="500">
+
+3. You will get a warning for using third-party service. Click on **Continue**.
+
+<img src="img/pbi_third_party_warning.png" width="500">
+
+4. Enter server value. Click on **OK**.
+
+<img src="img/pbi_connection_string_options.png" width="500">

```

Comment:
As of PR#16 this dialog is changed.

Reply:
I've updated this screenshot to reflect changes from PR#16